### PR TITLE
Minor tweaks to loadout categories for 2 items

### PR DESCRIPTION
**Pull request by plsleavemealon**
## About The Pull Request
Allows you to choose some items from the other section in the accessory section in loadout.  This does not remove them from the "other" category, it just also includes them in the "accessory" category so people have more options available to them.

Specifically, adds the pocket watch and the insignia of steel to the accessory category, so people can choose to take them there instead.  Both items are already jumpsuit accessories so it felt fitting, and more options is always nice.

## Why It's Good For The Game
More options for people to better customize and tailor their character to suit their needs.

## Proof Of Testing
I have also tested this locally and it adds them in properly, however it does force them to be taken in both categories.  I'll see if I can figure out how to fix this but I am not sure how to at this time unfortunately :(
## Changelog

🆑
add: Added 2 new entries to the accessory category in loadouts.
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/loadout/categories/accessories.dm [hunk 1]
```diff
@@ -1 +1,8 @@
-// THIS IS WHERE LOADOUT DATUMS GO FOR ACCESSORIES (REMOVE THIS COMMENT AND TICK THIS FILE IF YOU ADD ANYTHING)
+/datum/loadout_item/accessory/pocketwatch
```
Comment (nikothedude) [inline modular_zubbers/code/modules/loadout/categories/accessories.dm:1]: did you tick the file? only one file changed

Comment (plsleavemealon) [reply modular_zubbers/code/modules/loadout/categories/accessories.dm:1]: I think so? I honestly haven't done anything like this before.

Comment (plsleavemealon) [reply modular_zubbers/code/modules/loadout/categories/accessories.dm:1]: This is my first ever pull request that I've ever made XD

Comment (nikothedude) [reply modular_zubbers/code/modules/loadout/categories/accessories.dm:1]: are you editing this in vscode? if so, hit the unticked button at the bottom right of the screen when on this file

if on DM, i dunno

Comment (plsleavemealon) [reply modular_zubbers/code/modules/loadout/categories/accessories.dm:1]: Am on DM, and its ticked on there....

Comment (nikothedude) [reply modular_zubbers/code/modules/loadout/categories/accessories.dm:1]: then its already ticked and this isnt an issue

Comment (nikothedude) [reply modular_zubbers/code/modules/loadout/categories/accessories.dm:1]: i think so

**Comment by plsleavemealon** [comment]
Hmm, unfortunately I need to convert this back to a draft as there is a bug with it.  If you choose the pocket watch or insignia in the accessory slot, it also selects the item in the "other" category.

**Comment by plsleavemealon** [comment]
Should I close this for now? I honestly don't know how to fix this bug with it, and it would likely require some tweaks to how loadouts function...which is probably beyond my abilities :(

**Pull request closed without merging**